frontend/pages/setting.vue

```diff
@@ -141,6 +163,7 @@
                   contain
                   aspect-ratio="1"
                   lazy-src="/images/index-logo-gray.png"
+                  @click="showImage(cell)"
```

Comment:
```suggestion
                  @click.stop="showImage(cell)"
```
dialogのactivator slotを使わないので、 `.stop` をつける必要があるような気がします。

Reply:
ご指摘ありがとうございます！